src/test/java/io/vertx/tests/contract/VendorSpecificJsonTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2023, SAP SE
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *
+ */
+
+package io.vertx.tests.contract;
```

Comment:
shouldn't this be in the .impl package now?

Reply:
It's not really necessary, because it's a public class exposed to the `io.vertx.tests` module.

Reply:
Yes, technically it is not required, but all the other test classes follow this pattern. So I wouldn't change it now and put it in the related package. Then it is also easier to find.

Reply:
@pk-work I moved that ... I think we are good to go now :-)